CMakeLists.txt

```diff
@@ -192,6 +192,8 @@ option(WITH_EXAMPLES "Whether to build examples" ON)
 option(WITH_METRICS_PREVIEW "Whether to build metrics preview" OFF)
 option(WITH_LOGS_PREVIEW "Whether to build logs preview" OFF)
 option(WITH_ASYNC_EXPORT_PREVIEW "Whether enable async export" OFF)
+option(WITH_METRICS_EXEMPLAR_PREVIEW
+       "Whethere to enable exemplar within metrics" OFF)
```

Comment:
Typo: Whether

Reply:
And mentioned the spec status is experimental, so user will be aware of that instead of considering it as implementation gap?